src/consensus/README.md

```diff
@@ -1,10 +1,12 @@
-In this document, we describe the necessary components comprising the Consensus layer of a Cardano blockchain node. The main goal of this report is to provide guidance to software engineers that intend to implement the Consensus layer of a Cardano node from scratch.
+# The Consensus and Storage Layers
 
-This document is a work in progress. We strive to provide a set of requirements for and responsibilities of the Consensus layer that is agnostic of a particular implementation. However, we still very much are informed by the Haskell implementation in [ouroboros-consensus](https://github.com/IntersectMBO/ouroboros-consensus) as it is the only one.
+In this document, we describe the necessary components comprising the Consensus and Storage layers of a Cardano blockchain node. The main goal of this report is to provide guidance to software engineers that intend to implement such layers of a Cardano node from scratch.
 
-# What does Consensus and Storage need to do: responsibilities and requirements
+This document is a work in progress. We strive to provide a set of requirements for and responsibilities of the Consensus layer that is agnostic of a particular implementation.
 
-## Responsibilities of the Consensus Layer
+The data stored by a Cardano block is precisely the data used and produced by the Consensus and Ledger layers, it is therefore reasonable to pair such layers together in a single component, although separate components could also be envisioned. Note however that the bandwith between such components is very high.
```

Comment:
I think this is a very important point, but I suggest we stay focused in this section. Perhaps we could move this paragraph somewhere more top-level?